packages/components/src/components/hds/app-frame/parts/footer.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export interface HdsAppFrameFooterSignature {
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLElement;
```

Comment:
Noting here that there is no `HTMLFooterElement` that we could use, unfortunately.